agent/api/task/task_windows.go

```diff
@@ -73,8 +73,23 @@ func (task *Task) downcaseAllVolumePaths() {
 	}
 }
 
+func isBareDrive(path string) bool {
```

Comment:
nit: I tend to prefer having the caller function above the callee function (so `getCanonicalPath` above `isBareDrive`) as I find it easier to read (I can understand why the function exists before reading the body of it).